contributors/devel/release/scalability-validation.md

```diff
@@ -0,0 +1,123 @@
+# Scalability validation of the release
+
+Scalability is an integral part of k8s. Numerous issues were identified during release 1.7 while running tests on large clusters (2k-5k nodes). A majority of these manifested only in large clusters - more info under umbrella issue [#47344]. The issues ranged from large cluster setup (both cloud-provider specific and independent ones) to test-infra problems to previously uncaught regressions with performance and resource usage and so on.
+
+We started [supporting] 5000-node clusters from k8s 1.6 and need to make sure this scale is supported in future releases too. We did it this time by running performance and correctness tests under different configurations manually. But this testing procedure needs to be automated, concrete & well-maintained.
+
+## Goals
+
+In this document, we address the following process-related problems wrt scale testing:
+
+- Automate large cluster tests with a reasonable frequency
+- Concretely define the testing configuration used for the release
```

Comment:
> Hmm, seems that your goals and non-goals are in conflict with each other. How can you concretely define a configuration without defining (or at least knowing) what tests, workloads and SLOs you are subject to?

@jeremyeder - I completely don't agree here. We know that we (will) have some set of performance tests we believe are what we are trying to set. And *all* of those are hidden under this "performance tests* term. Defining those tests is unrelated to the effort of saying how important they are (which is more-or-less this doc).